### ISSUE TITLE
support large size png

### ISSUE THREAD

**Issue by michaelchao** [eXeC64/imv#197]
This project is awesome!
I modified your project so that it will use libpng to show PNG picture, and I found that if I use it to show PNG which is large size, for example, 22M, this kind of PNG will not be shown.
So I read more post about libpng to show large size. This one will resolve it:
`
png_set_chunk_malloc_max(private->png, 1024 * 1024 * 50); // max size is 50M for now :)
`

at :

> src/backend_libpng.c

This is my idea, thank you for you awesome project again:)

**Comment by rien333** [comment]
I ran into this as well, but under some circumstances large pngs will still open. 

On my system,  strangely enough,  `imv`  only fails to load large pngs when it's called through `xdg-open` (e.g. via the `.desktop` file). If call `/usr/bin/imv` or  `/usr/bin/imv-x11` directly, the image will open quite quickly. If I modify the `.desktop` file  to call `imv-x11` instead of `imv`, `xdg-open` will also work correctly. 

[cover.zip](https://github.com/eXeC64/imv/files/3664631/cover.zip)

As a reference, I've included the file in question. It should be noted that other image viewers (`sxiv`, `mpv`, my browser) open this file without problems.
